app/controllers/groups/group_links_controller.rb

```diff
@@ -0,0 +1,43 @@
+# frozen_string_literal: true
+
+module Groups
+  # Controller actions for Group Group Links
+  class GroupLinksController < Groups::ApplicationController
+    include ShareActions
+
+    def group_link_params
+      params.require(:namespace_group_link).permit(:group_id, :group_access_level, :expires_at)
+    end
+
+    private
+
+    def namespace_group_link
+      @namespace_group_link ||= NamespaceGroupLink.find_by(id: params[:id])
```

Comment:
**nitpick (blocking):** We should probably change this query to be scoped to the `Group`, so that might look like `@group.shared_with_group_links.find_by(id: params[:id])`.

The idea being here, that we don't want to allow editing a NamespaceGroupLink in a place where it should not be editted.

Reply:
Good idea! Updated in 07cfcddc1c8ffc87c49116e40478b27068c0ee56